src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,12 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
+        String clean = text.replaceAll("[\\W]", "").toUpperCase();
+        String reverseString = new StringBuffer(clean).reverse().toString();
+        System.out.println(reverseString);
+        if (clean.equals(reverseString)) {
+            return true;
+        }
```

Comment:
what does equals() return?

Reply:
returns true or false, understood what you mean, resolve soon and send you back